utils.go

```diff
@@ -20,6 +20,9 @@ var (
 
 // repeats a space n times
 func s(n int) string {
+	if n <= 0 {
```

Comment:
if we change this to `if n < 0 {`, all changes to `fmt_pretty.go` can be removed

Reply:
Yes but it needs another change on the code with the same check I initially have to have a space before comments. Not sure witch one is the better option

Reply:
All the testcases still pass with my suggestion, do you have a scenario where it wouldn't work?

Reply:
it also seems like the new TC doesn't brake when utils.go is reset to master

Reply:
😕 I initially add the issue on the injection test, but as it seems to behave cleanly now, it probably is useless now ~~ will clean it 👍

Reply:
Ok. I get what happened here. 
I initially add  `	allowInjection` variable sat to false when I made the first test cases but then it broke the concurrency run. So I tried using allowInjection as true by default. Seems that doing it render the test useless and it injected the step earlier than expected. Putting it back to false render the test meaningfull again. But i'll probably have to fix again the concurrency runs.